### ISSUE TITLE
Missing definition in Understanding of 1.2

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag#2956]
At the understanding of [Time-based Media](https://www.w3.org/WAI/WCAG21/Understanding/time-based-media) the following sentence is incomplete

> Synchronized media is defined in the glossary as:

Missing link to https://www.w3.org/TR/WCAG21/#dfn-synchronized-media

**Comment by alastc** [comment]
In the source code (pre-build) we have:

```
<p>Synchronized media is defined in the glossary as:</p>    
<p id="copysynchronizedmediadef"></p>
```

@michael-n-cooper I'm not familiar with that, is it supposed to be an 'include'?

**Comment by michael-n-cooper** [comment]
@alastc yes that was an include processed by the 2.0 xslt, I resolved many of those in the port but guess overlooked this one.

**Comment by kfranqueiro** [comment]
This was fixed in #4262.
